### ISSUE TITLE
Provide app's appdata problems in the api

### ISSUE THREAD

**Issue by jgarciao** [flathub/linux-store-backend#23]
[UpdateServiceImpl](https://github.com/flathub/linux-store-backend/blob/e72fd22275fa1ac4a95a37bb702aafa01458c4c6/src/main/java/org/flathub/api/service/UpdateServiceImpl.java#L248) print in the logs the appdata problems found in some apps by [AppdataValidator](https://github.com/flathub/linux-store-backend/blob/master/src/main/java/org/flathub/api/util/AppdataValidator.java), like missing icons, etc.

It would be useful add this info in the api, so it could be shown in a specialized page in the website to help developers fix the problems with their apps

Related to #21

**Comment by nedrichards** [comment]
This is probably obsoleted by support for the validator in the build process.

**Comment by barthalion** [comment]
Not exactly. FontForge failed to display because had too long and too many per-language `<p>` descriptions and we still need to add check for icon availability in buildbot (svg/png with correct sizes).
